bridge_devices/sonoff_433rf_bridge/firmware.lua

```diff
@@ -0,0 +1,293 @@
+-- Configuration variables must be also defined
+-- in `write_configuration` command arguments in manifest.yml
+IP_ADDRESS_CONFIG = 'ip_address'
+IP_PORT_CONFIG = 'ip_port'
+DEVICEID_CONFIG = 'device_id'
+REMOTE_NUMBER_CONFIG = 'remote_number'
+
+json = require("json")
```

Comment:
I think you can safely move this line to `get_data()`. don't forget to make it `local`